requirements.txt

```diff
@@ -1 +1,3 @@
 -e .
+setuptools
```

Comment:
These requirements should be added to the main section of `setup.py`.  If we have a `setup.py` file then the `requirements*.txt` files should just call that so we have a single source of truth.

Reply:
I think `wheel` should remain in `requirements.txt` because it is an environment _preference_ and not something required for the project to function. Per [packing documentation](https://packaging.python.org/discussions/install-requires-vs-requirements/#install-requires-vs-requirements-files):

> `install_requires` is a setuptools `setup.py` keyword that should be used to specify what a project minimally needs to run correctly.

Reply:
I removed `setuptools` from `requirements.txt` in 370f2a2.

Reply:
Can @felddy, @dav3r, and @hillaryj weigh in here?  We should do the same thing everywhere, and @felddy and I added `wheel` to `setup.py` in cisagov/development-guide#21.

Reply:
I have no preference here.  @mcdonnnj makes a good point about `wheel` being a preference vs. being required, but I'm not sure it makes a big difference either way in this case.  I'll go with whatever the group decides on this one.

Reply:
@mcdonnnj makes a good point, but I also like having a single source of truth.  I think mixing dependencies between `requirements*.txt` and `setup.py` files will eventually lead to confusion.  I think simplicity trumps his argument in this case, but I will also concede to whatever the group decides.

Reply:
I would like to point out that we already do this with `requirements-dev.txt` where `ipython` and `semver` are described as preferences for a development environment. They are not necessary for the functionality of the package, but are used for development.

Reply:
OK, you have convinced me.  Can you create a PR to do the same thing in [cisagov/development-guide](https://github.com/cisagov/development-guide)?

Reply:
I approve this train of... thought.  🧠 🚂 
ALL ABOARD THE BRAIN TRAIN!!! WOO WOO!